autosklearn/ensembles/abstract_ensemble.py

```diff
@@ -1,18 +1,29 @@
+from __future__ import annotations
+
 from abc import ABCMeta, abstractmethod
-from typing import Dict, List, Tuple, Union
+from typing import Dict, List, Sequence, Tuple, Union
 
 import numpy as np
 
+from autosklearn.metrics import Scorer
 from autosklearn.pipeline.base import BasePipeline
 
 
-class AbstractEnsemble(object):
+class AbstractEnsemble:
     __metaclass__ = ABCMeta
```

Comment:
Can remove this `__metaclass__ = ABCMeta`, it's more or less equivalent to `class AbstractEnsemble(ABC):` which modern tooling has better support for.

Reply:
Done